src/GoTrueAdminApi.ts

```diff
@@ -213,7 +223,7 @@ export default class GoTrueAdminApi {
    *
    * This function should only be called on a server. Never expose your `service_role` key in the browser.
    */
-  async deleteUser(uid: string): Promise<UserResponse> {
+  private async deleteUser(uid: string): Promise<UserResponse> {
```

Comment:
hmm why is this private?

Reply:
Yup, good catch — It’s not meant to be private—  was testing something and forgot to change back. Let me also delete the unused admin functions